app/checklist/views.py

```diff
@@ -88,3 +88,16 @@ def delete_default_checklist_item(id):
         flash('Error occurred. Please try again.', 'form-error')
         return redirect(url_for('checklist.index'))
     return redirect(url_for('checklist.index'))
+
+
+@checklist.route('/<int:id>/view', methods=['GET'])
```

Comment:
nit: let's change `<int:id>` to `<int:user_id>` so there's less confusion

Reply:
Got it!